tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -3034,47 +3034,47 @@ public function userRestoresVersionIndexOfFile($user, $versionIndex, $path) {
 	}
 
 	/**
-	 * @Then /^the (?:propfind|search) result of "([^"]*)" should (not|)\s?contain these files:$/
+	 * @Then /^the (?:propfind|search) result of "([^"]*)" should (not|)\s?contain these entries:$/
```

Comment:
make it more universal, so old steps don't break
```suggestion
	 * @Then /^the (?:propfind|search) result of "([^"]*)" should (not|)\s?contain these (?:files|entries):$/
```